interface/patient_file/encounter/forms.php

```diff
@@ -584,7 +584,7 @@ function findPosY(obj) {
 
     // Check for no access to the encounter's sensitivity level.
     $result = sqlQuery("SELECT sensitivity FROM form_encounter WHERE pid = ? AND encounter = ? LIMIT 1", [$pid, $encounter]);
-    if (($result['sensitivity'] && !AclMain::aclCheckCore('sensitivities', $result['sensitivity'])) || !$authPostCalendarCategory) {
+    if (($result['sensitivity'] && !AclMain::aclCheckCore('sensitivities', $result['sensitivity'])) || (!$authPostCalendarCategory ?? '')) {
```

Comment:
I almost fixed this when doing some php8, but it just seemed odd that it was using a variable that did not exist yet, so left it with plan to figure out why that was so. But then I forgot about it. Doesn't it seem odd that the variable is declared below this and used in other comparisons (ie. why is this variable even used here?). Perhaps this resulted from some sort of refactoring that has brought in a bug.